pzsd_bot/settings.py

```diff
@@ -41,5 +42,48 @@ class _DB(EnvConfig):
 
 DB_CONNECTION_STR = f"postgresql+asyncpg://{DB.pguser}:{DB.pgpassword}@{DB.pghost}:{DB.pgport}/{DB.pgdatabase}"
 
+
+class _PointsSettings(EnvConfig):
```

Comment:
Badly named

Reply:
ParsingSettings

Reply:
ParsingConstants

Reply:
I didn't love it either, but I couldn't name it `_Points` like I originally wanted to without renaming the `Points` class in points.py. And I can't think of an alternate name. Suggestions?

Reply:
ParsingConstants

Reply:
Not a fan of these names lol. I'm not opposed to renaming in the future, but until we come up with a better one, I'm keeping it the same for now.